less/styles.less

```diff
@@ -21,6 +21,10 @@
   background-position: 0 100%;
   .featured-jumbotron{
     padding-top:48px;
+
+    .btn {
+      border-radius: 1em;
```

Comment:
Instead of updating the border radius manually in CSS you should update Bootstrap's variable: @border-radius-base.

Reply:
it's already here
https://github.com/flora8984461/ecdtools.eclipse.org/blob/yiliu/master/makeFeaturedStoryButtonRound/less/_variables.less#L37

But for `.featured-jumbotron. btn` it has `border-radius: 0` on small size screen, I think there is no variable for it? 🤔

Reply:
I update it to use the variable in the styles.less instead of "1em"

Reply:
Ah ok I didn't think we were overriding it for the Jumbotron. No I don't think we would have created a variable for that. I would've thought that all the buttons on the website would have the same border radius for consistency?

Reply:
Yeah, I thought so as well, but when you view ecdtools on mobile screen size, I find this style and makes it square.
![image](https://user-images.githubusercontent.com/39588094/100628273-06103080-32f6-11eb-8217-e8df7f343e1e.png)